opencog/python/web/api/apiatomcollection.py

```diff
@@ -51,7 +51,7 @@ def __init__(self):
         self.reqparse.add_argument('limit', type=int, location='args')
 
         super(AtomCollectionAPI, self).__init__()
-        self.atomspace = opencog.cogserver.get_server_atomspace()
+        # self.atomspace = opencog.cogserver.get_server_atomspace()
 
```

Comment:
That won't be necessary. Passing an atomspace to the constructor seems like the optimal way to do it but AtomCollectionAPI receives the default AtomSpace through it's new class method. Cosmo has explained why he chose this approach: https://github.com/twilio/flask-restful/issues/134